lib/rialto/etl/cli/error_reporter.rb

```diff
@@ -7,16 +7,12 @@ module Etl
     module CLI
       # Logs errors to STDERR and to Honeybadger
       class ErrorReporter
-        extend Logging
 
         # Set up Honeybadger manually, so that we set the logger before honeybadger
         # logs its first message.
         Honeybadger.init!(framework: :ruby, env: ENV['RUBY_ENV'])
         Honeybadger.load_plugins!
         Honeybadger.install_at_exit_callback
-        Honeybadger.configure do |config|
-          config.logger = logger
```

Comment:
If you do this, will HB log to STDOUT?

Reply:
If so, does that cause the HB logs to be redirected into the JSON?

Reply:
I think I need to add a honeybadger.log file to this, I did that locally and it logged there fine.